dev/pubspec.lock

```diff
@@ -0,0 +1,369 @@
+# Generated by pub
```

Comment:
Do we really need all these packages to validate a json file or we need an updated file?

Reply:
removed unnecessary dependencies, and re-ran `pub get`. Not much change though.